### PR TITLE
added explain page for results and early stopping. added new method "…

### PR DESCRIPTION
**Pull request by LinoGiger**
…new_complex_order" for the deprecated "new_order"

### CODE REVIEW

File: openapi/output.swagger.json
Comment (Makuh17) [inline openapi/output.swagger.json:1]: What is this file?

File: src/rapidata/rapidata_client/rapidata_client.py [hunk 1]
```diff
@@ -54,10 +54,24 @@ def __init__(
...
 
+        Args:
+            name (str): The name of the order.
+
+        Returns:
+            RapidataOrderBuilder: A RapidataOrderBuilder instance.
+        """
+        return self.new_complex_order(name)
+    
+    def new_complex_order(self, name: str) -> RapidataOrderBuilder:
```
Comment (Makuh17) [inline src/rapidata/rapidata_client/rapidata_client.py:69]: Same as above, why no advanced order?

File: src/rapidata/rapidata_client/selection/capped_selection.py [hunk 2]
```diff
@@ -21,5 +21,5 @@ def to_model(self):
                 CappedSelectionSelectionsInner(selection.to_model())
                 for selection in self.selections
             ],
-            max_rapids=self.max_rapids,
+            maxRapids=self.max_rapids,
```
Comment (Makuh17) [inline src/rapidata/rapidata_client/selection/capped_selection.py:24]: Why this change?

**Review by Makuh17 — APPROVED** [review]

File: openapi/output.swagger.json
Comment (LinoGiger) [reply openapi/output.swagger.json:1]: good question, i didn't intentionally touch this. @kannwism do you know why this might change?

File: src/rapidata/rapidata_client/selection/capped_selection.py (hunk 2 above)
Comment (LinoGiger) [reply src/rapidata/rapidata_client/selection/capped_selection.py:24]: it's something i encountered. the backend changed it and i guess we never did realize it was broken

File: docs/advanced_orders.md [hunk 3]
```diff
@@ -28,7 +28,7 @@ Each request to humans begins with creating an order. Use the `rapi` client to c
 1. Create a new [`RapidataOrderBuilder`](reference/rapidata/rapidata_client/order/rapidata_order_builder.md/#rapidata.rapidata_client.order.rapidata_order_builder.RapidataOrderBuilder) and specify the name:
 
 ```py
-order_builder = rapi.new_order("Example Custom Order")
+order_builder = rapi.new_complex_order("Example Custom Order")
```
[the remark this answers is not in the document]
Comment (LinoGiger) [reply docs/advanced_orders.md:31]: i took it from the task luca gave me. idm changing it to advanced

File: openapi/output.swagger.json
Comment (kannwism) [reply openapi/output.swagger.json:1]: This is the combined swagger api used by the generator. Once can generate the whole generated client stuff from it, that's why it's in version control. But there should only be changes on this file in PRs created by the backend. 

The current status of the PR does not seem to include changes here, so all good I'd say.

File: docs/confidence_stopping.md [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+The Early Stopping feature leverages the trustworthiness, quantified through their `userScores`, to calculate the confidence level of each category for any given datapoint.
+
+### Confidence Calculation
+- **UserScores**: Each annotator has a `userScore` between 0 and 1, representing their reliability. [More information](/rapidata-python-sdk/understanding_the_results/#understanding-the-user-scores)
+- **Aggregated Confidence**: By combining the userScores of annotators who selected a particular category, the system computes the probability that this category is the correct one.
+- **Threshold Comparison**: If the calculated confidence exceeds your specified threshold, the system stops collecting further responses for that datapoint.
+
+## Understanding the Confidence Threshold Plot
+
+To help you estimate the number of responses needed to reach different confidence levels, we've conducted a thousand simulations based on actual userScore distributions.
```
[the remark this answers is not in the document]
Comment (Makuh17) [reply docs/confidence_stopping.md:27]: I honestly wouldn't even mention the simulations. All the customer needs to know is that they can use the plot to get an idea of what to expect. (I might be alone with this opinion though) Alternatively you could write "based on simulations"
